extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -323,22 +209,15 @@ These are methods that should be implemented:
 - `fullscreenenter`
 - `fullscreenexit`
 
-You can choose to only implement this interface partially, with a few caveats:
+Enabling playback [actions](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events/) require the `play` and `pause` methods. 
```

Comment:
```suggestion
Enabling playback [actions](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events/#video-elements) require specific methods:

- `element.play`: `method('play', ...)`
- `element.pause`: `method('pause', ...)`
- `element.mute`: `method('mute', ...)`
- `element.unmute`: `method('unmute', ...)`
- `element.fullscreen`: `method('fullscreenenter', ...)`
```

I dislike that we need a mapping table only because `fullscreen` is different from `fullscreenenter`. Alternatively we can alias `fullscreenenter` to `fullscreen` on the `amp-video-iframe` implementation. I'd be happy to create consistency between action and method names so we can avoid the table altogether.